app/assets/javascripts/modules/media_viewer.js

```diff
@@ -146,11 +147,22 @@
       return hlsSource.prop('src').indexOf('/mp3:') >= 0;
     }
 
+    function ms10Browser() {
+      var fingerprinter = new ClientJS();
```

Comment:
We should add the comment I describe to let hound know that a global is declared elsewhere that we will be using in this file (as is the established pattern).